### ISSUE TITLE
spotless maven eclipse jdt - How to block import star (*) wildcard?

### ISSUE THREAD

**Issue by tivrfoa** [diffplug/spotless#832]
Hi!

Spotless is really cool project!

I am using maven plugin with eclipse jdt.

Is there a way to block `import package.*;` please?

I think there is no way (currently) to remove package.* and put the correct imports, but I would like to at least it to give an error, so the developer can fix it.

Thank you!! :)

**Comment by smoothreggae** [comment]
This sounds similar to #649. [One of the comments](https://github.com/diffplug/spotless/issues/649#issuecomment-738199681) has something you could try as an alternative.

**Comment by nedtwigg** [comment]
Thanks @smoothreggae!

**Comment by tivrfoa** [comment]
> This sounds similar to #649. [One of the comments](https://github.com/diffplug/spotless/issues/649#issuecomment-738199681) has something you could try as an alternative.

Thank you!

Where should the code below be placed in pom.xml please?

```xml
              <replaceRegex>
                <name>Remove wildcard imports</name>
                <searchRegex>import\s+[^\*\s]+\*;(\r\n|\r|\n)</searchRegex>
                <replacement>$1</replacement>
              </replaceRegex>
```

I placed inside `<java></java>`, but `mvn spotless:check` didn't spot it :]
I'm using spotless version 2.4.1

```xml
					<java>
						<importOrder>
							<order>br,com,java,javax,org,</order>
						</importOrder>
						<removeUnusedImports />
						<replaceRegex>
								<name>Remove wildcard imports</name>
								<searchRegex>import\s+[^\*\s]+\*;(\r\n|\r|\n)</searchRegex>
								<replacement>$1</replacement>
							</replaceRegex>
						<eclipse>
							<version>4.13.0</version>
							<file>${basedir}/eclipse-formatter.xml</file>
						</eclipse>
					</java>
```

**Comment by tivrfoa** [comment]
I updated the text. It worked! Thanks again!

**Comment by nedtwigg** [comment]
Released in `plugin-gradle 7.1.0` and `plugin-maven 2.45.0`.
